src/WebJobs.Extensions.DurableTask/EntityScheduler/RequestMessage.cs

```diff
@@ -12,13 +12,25 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
     /// </summary>
     internal class RequestMessage
     {
-        private readonly MessagePayloadDataConverter dataConverter;
+        // This no-arg constructor is used when a RequestMessage is deserialized.
+        public RequestMessage() { }
```

Comment:
I was looking at this issue as well. I have an alternate fix that I can create as a PR if you're interested. It might be a bit simpler.